src/__tests__/App.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../components/App";
+
+describe("App", () => {
```

Comment:
maybe put in a bit more detail in labelling your describe block. Even if it's just 'App tests' (this is a bit nitpicky but it always helps to name things clearly)